test/TestBasics.cpp

```diff
@@ -220,6 +229,7 @@ TEST_CASE("Mocxx::Replace() system functions", "[Mocxx]")
 
 TEST_CASE("Mocxx::ReplaceMember()", "[Mocxx]")
 {
+#if !defined(_WIN32) || defined(_WIN64)
```

Comment:
Can you elaborate on why this test is disabled?

Reply:
Of course!  In 64-bit this test works as intended:
<image src="https://user-images.githubusercontent.com/3475163/104361496-85417880-54e0-11eb-89df-0046c0b38fd3.png" width="75%" />

But in 32-bit AKA x86 it doesn't seem to work for Windows.  Here is a Gif showing what I am getting:
![x86Test](https://user-images.githubusercontent.com/3475163/104361606-ae620900-54e0-11eb-9a70-38cca5355b9b.gif)

On MSVC here is a link to predefined macros: https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-160

`_WIN32` doesn't mean 32-bit but also is defined if you are building 64-bit on Windows.  `_WIN64`, however, is only defined when building 64-bit on Windows.  So this test is just disabled for 32-bit Windows.

I would like to **not** disable this nor any other test.  Since it does work in 64-bit on Windows it is a tad odd that 32-bit on Windows has a failure.  I did try a couple of disabling of optimization settings and it continued to fail.  I do believe there is a setting I still need to configure to turn off an optimization such that this can work.

Hope that makes sense.  Holler if there is more information I can provide.  And I will be continue to try things to see if I can figure this out.